supernode/daemon/daemon.go

```diff
@@ -50,7 +50,7 @@ func (d *Daemon) RegisterSuperNode() error {
 	req := &types.PeerCreateRequest{
 		IP:       strfmt.IPv4(d.config.AdvertiseIP),
```

Comment:
@lowzj  FYI. For now, we only support the local storage file server. In the future, we should make it possible to config the IP the remote file server.